src/styles/_media.scss

```diff
@@ -0,0 +1,17 @@
+@media (min-width: 488px) {
+  .catalog {
+    grid-template-columns: repeat(2, 200px);
+  }
+}
+
+@media (min-width: 768px) {
+  .catalog {
+    grid-template-columns: repeat(3, 200px);
+  }
+}
+
+@media (min-width: 1024px) {
+  .catalog {
+    grid-template-columns: repeat(4, 200px);
```

Comment:
do not create separate file for media, move it to the catalog.scss

Reply:
got it